src/tasks/containers/TasksPage.test.tsx

```diff
@@ -282,10 +282,6 @@ describe('Tasks.Containers.TasksPage', () => {
 
       expect(mocked(parse).mock.calls[0][0]).toEqual(sampleScript)
 
-      expect(mocked(postTask).mock.calls[0][0].data.id).toEqual(InactiveTask.id)
-      expect(mocked(postTask).mock.calls[0][0].data.name).toEqual(
-        InactiveTask.name
-      )
```

Comment:
The method call that this is spying on sends `data.id` and `data.name`, but `postTask` doesn't do anything with them, and the type for the parameters to `postTask` doesn't list `id` and `name`, which generated a type error.